src/backend/cuda/fft.cpp

```diff
@@ -25,11 +27,15 @@ using std::string;
 namespace cuda
 {
 
+typedef std::pair<string, cufftHandle> FFTPlanPair;
+typedef std::deque<FFTPlanPair> FFTPlanCache;
```

Comment:
Hmm, but then it might get hard to figure out which plan top pop. Ok, let's leave this in place.